### PR TITLE
lua works, but executes the tests four times. Don't know why.

### PR DESCRIPTION
**Pull request by ciscoheat**

### CODE REVIEW

File: src/travix/Travix.hx [hunk 1]
```diff
@@ -619,6 +668,9 @@ class Travix {
       case 'php': t.doPhp();
       case 'python': t.doPython();
       case 'cs': t.doCs();
+      case 'lua': 
+        if(haxeVersion < "3.3.0") die('lua requires haxe >= 3.3.0');
```
Comment (back2dos) [inline src/travix/Travix.hx]: Wouldn't it be preferable if the test were skipped? Then `all` could always add `lua` and you'd get your results on haxe versions where it's supported. What's your rationale for raising an error?

Comment (ciscoheat) [inline src/travix/Travix.hx]: Maybe so, my rationale is that if I'm trying to run lua tests and it's not supported, I'd like to know what's going on.

Comment (back2dos) [inline src/travix/Travix.hx]: Understandable, although if you can't generate lua code to start with, there's nothing to be run. But you're right, such details would better be discussed when it's working ;)

**Comment by ciscoheat** [comment]
All right! :) So if that's fine, the only issue is that the tests are executing multiple times. I checked and it's only exiting one time, so it must be something lua-related. The tests are passing anyway, so I'll leave this open, if you want you can merge.

**Comment by back2dos** [comment]
Well then, let's summon @jdonaldson ... anything ideas why rather trivial unit tests might run multiple times?

**Comment by jdonaldson** [comment]
Hmmm... no idea.  The sys/process support is still very rough (and missing in some cases).  That may have something to do with it. 

Is there a "test" travis instance I can play around with?  Something I can run locally?

**Comment by back2dos** [comment]
Uhm, you can clone this repo and `haxelib dev travix . && haxelib run travix install` whereafter `haxelib run travix lua` should basically run on your machine what we're running on travis (+/- the usual differences).

Or `haxelib install travix && haxelib run travix init` in some folder, and then `haxelib run travix lua` in there, if you want to try it on different code.

**Comment by nadako** [comment]
any news on this one? it would be awesome to have lua tested!

**Comment by jdonaldson** [comment]
Not yet... it's second in my queue.  I need to update the externs for 5.1/5.2 support, and then sort out the version switching mechanism in travis.

**Comment by jdonaldson** [comment]
Can you add me to the project here?  I'll make sure to track these.

**Comment by jdonaldson** [comment]
thanks

**Comment by ciscoheat** [comment]
closed by #40

**Pull request closed without merging**